test/infrastructure/docker/main.go

```diff
@@ -207,9 +231,7 @@ func setupReconcilers(ctx context.Context, mgr ctrl.Manager) {
 		if err := (&expcontrollers.DockerMachinePoolReconciler{
 			Client:           mgr.GetClient(),
 			ContainerRuntime: runtimeClient,
```

Comment:
```suggestion
			ContainerRuntime: runtimeClient,
			Tracker:          tracker,
```
Please still pass in the tracker

Reply:
(via the struct, not the method)